src/components/AppProvider/tests/AppProvider.test.tsx

```diff
@@ -16,7 +16,7 @@ describe('<AppProvider />', () => {
   });
 
   it('updates context when props change', () => {
-    const Child: React.SFC<{}> = () => {
```

Comment:
A lot of the changes in this PR are "that's the default value for the generic, you don't need to specify that"